src/arb/arb_poly.jl

```diff
@@ -748,7 +748,8 @@ for T in [fmpz, fmpq, Float64, BigFloat]
    end
 end
 
-(a::ArbPolyRing)(b::Array{T, 1}) where {T <: Integer} = a(map(base_ring(a), b))
+
+(a::ArbPolyRing)(b::Array{T, 1}) where {T <: Union{ArbField,Integer}} = a(map(base_ring(a), b))
```

Comment:
Is there an example which fails without this change?

Reply:
Yes. See line 775.

Reply:
This was working also before:
```
julia> R = ArbField(64)
Real Field with 64 bits of precision and error bounds

julia> Rx, x = R["x"]
(Univariate Polynomial Ring in x over Real Field with 64 bits of precision and error bounds, [ 0, 1.0000000000000000000 ])

julia> Rx([R(1), R(2)])
[ 1.0000000000000000000, 2.0000000000000000000 ]
```
You defined a method for an array of `ArbField`, that is, `Rx([R, R])`.

Reply:
Indeed. I must have missed this. I will revert it.